LibreHardwareMonitorLib/Hardware/Gpu/AmdGpu.cs

```diff
@@ -759,18 +869,46 @@ public override string GetReport()
             r.AppendLine();
             try
             {
-                var data = new AtiAdlxx.ADLPMLogDataOutput();
-                AtiAdlxx.ADLStatus status = AtiAdlxx.ADL2_New_QueryPMLogData_Get(_context, _adapterInfo.AdapterIndex, ref data);
+                if (_usePmLog && _pmLogStarted) {
```

Comment:
This lot as well (`{` on new line, and same with else statement below. Also space after `for` below).

Reply:
I'm not used to this code style, I more a java developer... :) I've checked others like this, should be good now.